packages/ses/README.md

```diff
@@ -137,6 +146,61 @@ c1.globalThis === c2.globalThis; // false
 c1.globalThis.JSON === c2.globalThis.JSON; // true
 ```
 
+The global scope of every compartment includes a shallow, specialized copy of
+the JavaScript intrinsics, omitting `Date.now` and `Math.random` since these
+can be used as covert communication channels between programs that have not
+been expressly introduced.
+However, a compartment may be expressly given access to these objects
+through the first argument to the compartment constructor or by assigning
+them to the compartment's `globalThis` after construction.
```

Comment:
How about: 
However, a compartment may be expressly given access to these objects:
* Through the first argument in the compartment constructor or
* By assigning them to the compartment's `globaThis` after construction.